app/views/welcome.scala.html

```diff
@@ -37,8 +37,8 @@ <h2>Why do you see this page?</h2>
       Play has invoked the <code>controllers.HomeController.index</code> method to obtain the <code>Action</code> to execute:
     </p>
 
-    <pre><code>def index = Action { implicit request: Request[AnyContent] =>
```

Comment:
I believe this is an example on how to get a `request` within an HTML?

Reply:
isn't it confusing for first timer ?  ( at-least for me :) ) 
I assumed  basic idea of welcome-page is to explain the code written in HomeController. then it should be in sync with code. 
we can add another method which actually sends some request parameter and utilizing it through request instance in Action.